Runner/Program.cs

```diff
@@ -184,30 +189,22 @@ private static ServiceProvider BuildDependencyInjection(IConfiguration config) =
                 .AddTransient<ValidationClient>()
                 .AddSingleton(provider =>
                 {
-                    var rules = new IValidationRule[]
-                    {
-                        provider.GetService<HasDescriptionRule>(),
-                        provider.GetService<HasReadmeRule>(),
-                        provider.GetService<HasNewestPtcsJenkinsLibRule>(),
-                        provider.GetService<HasNotManyStaleBranchesRule>(),
-                        provider.GetService<HasLicenseRule>(),
-                        provider.GetService<HasCodeownersRule>()
-                    };
+                    var rules = selectedRules.Select(r => (IValidationRule)provider.GetService(r)).ToArray();
                     return new RepositoryValidator(
                         provider.GetService<ILogger<RepositoryValidator>>(),
                         provider.GetService<IGitHubClient>(),
                         rules);
                 })
                 .AddTransient<GitUtils>()
                 .AddTransient<DocumentationFileCreator>()
-                .AddTransient<HasDescriptionRule>()
-                .AddTransient<HasLicenseRule>()
-                .AddTransient<HasNewestPtcsJenkinsLibRule>()
-                .AddTransient<HasNotManyStaleBranchesRule>()
-                .AddTransient<HasReadmeRule>()
-                .AddTransient<HasCodeownersRule>()
                 .BuildServiceProvider();
 
+            var logger = provider.GetService<ILogger<Program>>();
+            logger.LogInformation($"Selected validation rules: {selectedRules}");
```

Comment:
Now that I'm checking, this part is already being done in `ValidationLibary.RepositoryValidator` constructor

Reply:
Indeed, I didn't see that. That is simpler.